app/pages/error/errorHandler.js

```diff
@@ -12,7 +10,7 @@ export const errorHandler = (err) => {
     formattedError.status = err.response.status;
     formattedError.message = `${err.response.data.errors[0]} ${err.response.statusText}`;
   }
-  formattedError.backLinkHref = `${baseUrl}/`;
+  formattedError.backLinkHref = '/';
```

Comment:
do we want this to be `/admin/organisations` - otherwise that will go back to publicbrowse

Reply:
Do we want to go back to public browse? This is for the generic error page so what about if it is  an authorisation based error? Do we handle that completely elsewhere?

Reply:
They are all good points and it highlights that we've not really been given much steer on when we do have errors. 
I think let's take them back to PB as you have done and if they want something different then that will have to be separate stories/tickets